src/bindings.cc

```diff
@@ -93,11 +93,11 @@ namespace NodeFuse {
         }
 
         fuse_session_add_chan(fuse->session, fuse->channel);
-        if(fuse->multithreaded){
-            ret = fuse_session_loop_mt(fuse->session); //blocks here
-        }else{
-            ret = fuse_session_loop(fuse->session); //blocks here            
-        }
+        // if(fuse->multithreaded){
```

Comment:
Good point.
I wasn't sure if I was going to revert back to threaded fuse, but threading really isn't needed with nodejs.